autopush/web/webpush.py

```diff
@@ -43,7 +44,9 @@ def post(self, api_ver="v1", token=None):
             headers=self.valid_input["headers"],
             ttl=self.valid_input["headers"]["ttl"]
         )
-
+        self._client_info["uaid"] = hasher(user_data.get("uaid"))
+        self._client_info["channel_id"] = user_data.get("chid")
+        self._client_info["stack"] = ["%s.post" % self.__module__]
```

Comment:
Until the request is validated, we don't have all the client info, such as uaid.